app/styleguide/styleguide.scss

```diff
@@ -0,0 +1,75 @@
+
+#sgLayout {
+  display: flex;
+  flex-direction: row;
+}
+
+
+#sgNav, #sgContent {
+  padding: 0.5rem 1rem;
+}
+
+#sgNav {
+  display: flex;
+  flex-direction: column;
+  border-right: 2px solid #e3e3e3;
+
+  a {
+    display: block;
+    padding: 0.5em 1em;
+  }
+}
+
+#sgContent {
+  & > div {
+    margin-bottom: 3em;
+    max-width: 35em;
+  }
+}
+
+.sg-example {
+  margin-bottom: 2rem;
+}
+
+.sg-button-list > * {
+  margin: 0.5em;
+}
+
+.brand-color-list {
+  > * {
+    margin-bottom: 1em;
+  }
+}
+
+ul.sg-spaced {
+  li {
+    margin-bottom: 1rem;
+  }
+}
+
+dd {
+  margin-top: var(--brand-space1);
+  margin-bottom: var(--brand-space2);
+}
+
+.brand-color {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  position: relative;
+  --_color: gray;
```

Comment:
1. You are right, But this was just a placeholder for a CSS variable to come once we have the designs and final colours. So it will be replace with a variable later.

2. `--_` is a pattern I use for reducing copy/paste code, while also specifying a "hidden" variable. It's not actually hidden, but it's an indicator that the variable _should not_ be overridden elsewhere in the CSS.

The use case for the latter is not clear because it's using a flat "gray" colour instead of the variable as stated in point 1. Here is a snippet of the intended use (this is from another project I maintain)
```
.product-list {
  --_gap: var(--product-list-grid-gap, 1rem);

  padding: var(--_gap);
  display: grid;
  grid-gap: var(--_gap);
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: auto;
}
```
As you can see, the original definition of the variable is quite long, and has a default case to use for this element. We also use it twice in the same element. Rather than repeat the `var(--product-list-grid-gap, 1rem)` every time we use it, we just use the shorter `--_gap` variable.

So the benefits for doing this I've found that,
1. Less mistakes are made because there is less duplication of the same code lines.
2. In future, when we need to change default behaviour, we don't have to remember to update every instance where it's used. In larger components with many states I found this to be quite helpful.

Wil all of the above said, if you want me to change it I will do so. It's a subjective code style after all :)

Reply:
Thanks for explaining the --_ usage @andrewebdev.  I see the benefit especially if you don't want it to be globally available, and just apply an override locally.

Reply:
I added notes regarding this in the styleguide now.